avocado/core/messages.py

```diff
@@ -134,6 +134,9 @@ class FinishMessageHandler(BaseMessageHandler):
 
     def handle(self, message, task, job):
         message.update(task.metadata)
+        # This parameter is required by the HTML plugin
+        message.update({'actual_time_start': job.actual_time_start})
```

Comment:
Hi @ana I think this is not working properly, because it makes every test start time the same as job start time, but the tests can have different start time. As an example:
`avocado run --nrunner-max-parallel-tasks 1 --test-runner=nrunner /bin/true timeouttest.py passtest.py 
`